src/panels/lovelace/components/hui-image.ts

```diff
@@ -172,27 +196,63 @@ export class HuiImage extends LitElement {
     `;
   }
 
+  protected _shouldStartCameraUpdates(oldHass?: HomeAssistant): boolean {
+    return !!(
+      (!oldHass || oldHass.connected !== this.hass!.connected) &&
+      this.hass!.connected &&
+      this.cameraView !== "live"
+    );
+  }
+
+  protected _shouldStopCameraUpdates(oldHass?: HomeAssistant): boolean {
+    return !!(
+      (!oldHass || oldHass.connected !== this.hass!.connected) &&
+      !this.hass!.connected
+    );
+  }
+
   protected updated(changedProps: PropertyValues): void {
     if (changedProps.has("hass")) {
       const oldHass = changedProps.get("hass") as HomeAssistant | undefined;
-      if (!oldHass || oldHass.connected !== this.hass!.connected) {
-        if (this.hass!.connected && this.cameraView !== "live") {
-          this._updateCameraImageSrc();
-          this._startUpdateCameraInterval();
-        } else if (!this.hass!.connected) {
-          this._stopUpdateCameraInterval();
-          this._cameraImageSrc = undefined;
-          this._loadError = true;
-        }
+
+      if (this._shouldStartCameraUpdates(oldHass)) {
+        this._startIntersectionObserverOrUpdates();
+      } else if (this._shouldStopCameraUpdates(oldHass)) {
```

Comment:
It think this can just be `!this.hass!.connected`?

Reply:
Yes. I think so. Adjusted